README.md

```diff
@@ -3,8 +3,9 @@
 [![Gem Version](https://badge.fury.io/rb/toxiproxy.svg)](https://badge.fury.io/rb/toxiproxy)
 [![Test](https://github.com/Shopify/toxiproxy-ruby/actions/workflows/test.yml/badge.svg)](https://github.com/Shopify/toxiproxy-ruby/actions/workflows/test.yml)
 
-`toxiproxy-ruby` `>= 1.x` is compatible with the Toxiproxy `2.x` series.
-`toxiproxy-ruby` `0.x` is compatible with the Toxiproxy `1.x` series.
+- `toxiproxy-ruby` `>= 3.x` is compatible with the Toxiproxy `2.6+` series.
```

Comment:
Just calling this out, I'll cut a new 3.0 version because `PATCH` is only available in Toxiproxy 2.6 so it may be a breaking change if folks are using an old version of the Toxiproxy server (like our CI which used 2.4). An alternative approach would be to detect toxiproxy server version and switch between PATCH/POST automatically which would make the change non-breaking.

Any thoughts?

Reply:
Agreed, WDYT of [4e2488f](https://github.com/Shopify/toxiproxy-ruby/pull/186/commits/4e2488fc1a627c2f5be9d3f07ae4492570db2639)